site/content/installation.md

```diff
@@ -1,38 +1,40 @@
 
 ## Installing Packages 
 
-Anaconda comes with a package management system called `conda`. To install a package, open your command prompt and run 
-
-`conda install package-name`
+We recommend using the package management tool `pip` that comes with Base Python. To install a package, you can run `pip install package-name` from the command line.
 
 Note that this must be done from the command line and **not** the Python session (in contrast to package management in R).
 
-Base Python has its own package management tool called `pip`. If a package is not available with `conda install`, you can try running 
+If you are working on a computer that already has Anaconda installed (and predates the TOS changes), you can still use `conda-forge` (the open-source alternative to `conda`) to install packages. To do this, you can run:
+- `conda config --add channels conda-forge`
+- `conda install package-name`
 
-`pip install package-name`
+## Running Code Interactively
+To run Python code in Quarto notebooks or IPython notebooks (.ipynb files), we recommend installing the Jupyter package. You can do this by running `pip install jupyter` from the command line.)
 
-You should always try running `conda install` first, as this will do a better job of resolving package dependency issues.
+We also recommend installing the following from the `Extensions` section of VS Code:
+- Python
+- Jupyter
+- Quarto
 
 ## Using Virtual Environments 
  
-To avoid running into version issues between multiple projects that rely on different Python versions or different versions of packages, you might want to try using virtual environments. A virtual environment is a way to create an isolated space so you can, for example, run Python 2.7 for one project and Python 3.7 for another on the same computer. `conda` includes robust support for creating and managing different environments. For more detail, see [this guide](https://ui-research.github.io/virtual-envs/) on using virtual environments at Urban or the official [conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html). 
\ No newline at end of file
+To avoid running into version issues between multiple projects that rely on different Python versions or different versions of packages, you might want to try using virtual environments. A virtual environment is a way to create an isolated space so you can, for example, run Python 2.7 for one project and Python 3.7 for another on the same computer. 
+
+We recommend using the `virtualenv` package, which can be installed by running `pip install virtualenv` on the command line. `virtualenv` includes robust support for creating and managing different environments. For more detail, see [this guide](https://ui-research.github.io/virtual-envs/) on using virtual environments at Urban or the official [virtualenv documentation](https://virtualenv.pypa.io/en/latest/).
```

Comment:
`virtualenv` looks like the best choice to me, but I'm admittedly not super familiar with alternatives like `pipenv`. To-do to update SWUG page with a `virtualenv` set of instructions instead of the conda ones.

Reply:
Whereas `venv` is simple to use - I'm doing it on another project - based on this tutorial: https://docs.python.org/3/tutorial/venv.html